### PR TITLE
[Cleanup] Adopt eslint vue/recommended: ARPA Reporter: upload views

### PR DESCRIPTION
**Pull request by jeffsmohan**
## Description
Brings ARPA reporter upload views into compliance with new linting rules.

- ./src/arpa_reporter/views/Upload.vue
- ./src/arpa_reporter/views/Uploads.vue

A few notes for the reviewer:

- Most of the diff is auto-fixed, so reviewing commit by commit should make things easier

## Screenshots / Demo Video
Screenshot 2024-03-30 at 10 05 53 PM
Screenshot 2024-03-30 at 10 05 59 PM

## Testing
Manually verified basic behavior of upload and upload list views. 

### Automated and Unit Tests
- [ ] Added Unit tests

### Manual tests for Reviewer
- [ ] Added steps to test feature/functionality manually

## Checklist
- [ ] Provided ticket and description
- [ ] Provided screenshots/demo
- [ ] Provided testing information
- [ ] Provided adequate test coverage for all new code
- [ ] Added PR reviewers

### CODE REVIEW

File: packages/client/src/arpa_reporter/views/UploadsView.vue [hunk 1]
```diff
@@ -6,43 +6,72 @@
...
       :columns="columns"
       :rows="rows"
       :group-options="groupOptions"
       :sort-options="{
         enabled: true,
         initialSortBy: defaultSortOrder
       }"
-      styleClass="vgt-table table table-striped table-bordered"
-      ref="uploadsTable"
+      style-class="vgt-table table table-striped table-bordered"
```
Comment (tzinckgraf) [inline packages/client/src/arpa_reporter/views/UploadsView.vue:27]: I can't find this syntax in the docs. I can only find the old syntax

https://xaksis.github.io/vue-good-table/guide/style-configuration/style-classes.html#style-classes

Comment (jeffsmohan) [reply packages/client/src/arpa_reporter/views/UploadsView.vue:27]: Good question — Vue actually treats the two syntaxes as synonymous, and translates either to the same underlying prop name. The lint rule simply standardizes our usage within templates to kebab case, as is the community style. 

Bit more about this all documented here: https://vuejs.org/style-guide/rules-strongly-recommended.html#prop-name-casing

Just to double-check my understanding, I added a new class `new-test-class` into this prop, then confirmed the table was rendered with the new class: 

Screenshot 2024-04-02 at 8 23 55 PM
Screenshot 2024-04-02 at 8 23 20 PM

**Comment by jeffsmohan** [comment]
@tzinckgraf Thanks for the review! Responded to your comment, fixed a merge conflict in .eslintrc, and it's back to you for re-review.

File: packages/client/src/arpa_reporter/views/UploadsView.vue (hunk 1 above)
Comment (tzinckgraf) [reply packages/client/src/arpa_reporter/views/UploadsView.vue:27]: Perfect, thank you. That explanation is great. Thanks for checking.

**Review by tzinckgraf — APPROVED** [review]

**Pull request merged**